client/app/bundles/momentsApp/components/Chart/Chart.jsx

```diff
@@ -17,30 +17,19 @@ const colorSchemes = ['#6D0839', '#66118', '#7F503F', '#775577', '#CCAADD'];
  * Renders a Chart Kick element.
  *
  * We wrap the element here in case we want to replace ChartKick with another library.
+ *
+ * @param props - Comes from your rails view.
  */
-// We keep the class otherwise our enzyme tests can't reference this component by name
-export default class Chart extends React.Component<chartShape, {}> {
-    props: chartShape;
-
-    /**
-     * @param props - Comes from your rails view.
-     */
-    render() {
-      return (
-        <div>
-          {this.props.chartType === 'Line' ?
-            <LineChart
-              xtitle={this.props.xtitle}
-              ytitle={this.props.ytitle}
-              data={this.props.data}
-              colors={colorSchemes}
-            /> : <AreaChart
-              xtitle={this.props.xtitle}
-              ytitle={this.props.ytitle}
-              data={this.props.data}
-              colors={colorSchemes}
-            />}
+const Chart = ({ chartType, ...props }: chartShape) => (
+  chartType === 'Line'
+    ? <LineChart {...props} colors={colorSchemes} />
+    : <AreaChart {...props} colors={colorSchemes} />
+);
+Chart.displayname = 'Chart';
```

Comment:
Ahh interesting, good to know that there is a way to expose this to enzyme tests.